mixer/pkg/cache/ttlCache.go

```diff
@@ -154,7 +154,13 @@ func (c *ttlCache) Get(key interface{}) (interface{}, bool) {
 
 func (c *ttlCache) Remove(key interface{}) {
 	c.entries.Delete(key)
-	atomic.AddUint64(&c.stats.Writes, 1)
```

Comment:
For lruCache we do record the stats when we Remove (inside the linkEntryAtTail method)

Reply:
I don't think that's the case. Can you point out the line you're thinking of?

Reply:
same issue as above.. you have removed that line and I was looking at the current code in gogland. that is why your view and my view of the code was different.